### ISSUE TITLE
[Feature Request] Filter for Tablist/context menu

### ISSUE THREAD

**Issue by Termuellinator** [mozilla/multi-account-containers#1778]
When using https://github.com/stoically/temporary-containers in automatic mode, many tab containers are created that clutter the addon-icon menu (where you can select "always open in" etc..) and the context menu -> "open in tab  container" list.

It would be nice to have an option to ignore/hide containers according to a filter (like " tmp* " for example) in these lists. I guess the context menu would be a direct firefox-request though?

**Comment by grahamperrin** [comment]
> Filter … addon-icon menu

▶ probably a duplicate of #1103

If so, please close. Thank you.

**Comment by gpoole** [comment]
@grahamperrin I don't think it duplicates #1103. The request is for a config option that will filter out the containers that temporary-containers creates, rather than a filter search box. The reason is that currently MAC shows temporary containers in the menu, but they're not useful to see and not used through the MAC addon, so they just clutter the space up:

![image](https://user-images.githubusercontent.com/2898433/142079453-cf16c879-ab91-490b-a9f8-fe209da8142b.png)

Aside from clutter, the temporary containers are also unintentionally synced (stoically/temporary-containers#371), so this feature request I think would also be handy if it was for a means of entirely ignoring containers including from sync and in the UI.

**Comment by gpoole** [comment]
This could be mostly resolved by expanding on https://github.com/mozilla/multi-account-containers/pull/2231 to use the same pattern setting to hide the containers from the popup as well.

Unfortunately since the context menu is not controlled by this extension it would not be possible to exclude them from that.

**Comment by MurzNN** [comment]
The https://github.com/mozilla/multi-account-containers/pull/2231 MR is merged, but I can't find the introduced option anywhere in the settings and even in the code: https://github.com/search?q=repo%3Astoically%2Fmulti-account-containers+syncExcludeRegExp&type=code

Is this feature removed from the current version of the extension?
